src/wwwroot/app/views/candidates/candidates.controller.js

```diff
@@ -189,45 +189,40 @@ export default function CandidatesController(
    };
 
    vm.isCandidateWasToogled = (candidateId) => {
-      let foundedCand = find(vm.selectedCandidates, (cand) => {
+      return !!find(vm.selectedCandidates, (cand) => {
          return cand === candidateId;
       });
-      if (foundedCand) {
-         return true;
-      } else {
-         return false;
-      }
    };
 
    vm.toogleAll = () => {
       vm.selectedCandidates = [];
-      if (vm.isAllToogled) {
+      if (vm.toogleContainer.isAllToogled) {
          forEach(vm.candidates.candidate, (candidate) => {
             candidate.isToogled = true;
             vm.selectedCandidates.push(candidate.id);
          });
       } else {
          forEach(vm.candidates.candidate, (candidate) => {
-            candidate.isToogled = false;
+            if (!includes(vm.candidateIdsAttachedToVacancy, candidate.id)) {
+               candidate.isToogled = false;
+            }
```

Comment:
candidate.isToogled = includes(vm.candidateIdsAttachedToVacancy, candidate.id); is not correct?